core-bundle/contao/themes/flexible/main.css

```diff
@@ -1867,6 +1867,12 @@ ul.sgallery li {
 }
 
 /* List wizard */
+.tl_listwizard, .tl_listwizard li {
```

Comment:
```suggestion
.tl_listwizard {
```

No need to apply this to the `li` elements, too, is there?

Reply:
The margin and padding needs to be reset on the `li` elements.

Reply:
Not on my system (Safari 16.2 on macOS Ventura).

<img width="675" alt="" src="https://user-images.githubusercontent.com/1192057/211361608-1b95dfb0-4642-4c18-a9c9-cda199c0e025.png">

In which browser on which OS does the issue occur for you?

Reply:
You are right, I must have remembered that wrong.